starknet-foundry/crates/forge/src/running.rs

```diff
@@ -69,7 +69,6 @@ pub(crate) fn run_from_test_case(
         runner: Some(runner),
         starknet_state: StarknetState::default(),
         string_to_hint,
-        blockifier_state: None,
```

Comment:
👀

Reply:
Yea, I don't know why it was there in the first place XD